features/git-town-config/setup.feature

```diff
@@ -19,7 +19,7 @@ Feature: Initial configuration
   Scenario: does not prompt for perennial branches if there is only the main branch
     Given I haven't configured Git Town yet
     When I run `git-town config setup` and answer the prompts:
-      | PROMPT                                     | ANSWER                      |
-      | Please specify the main development branch | [DOWN][ENTER]               |
+      | PROMPT                                     | ANSWER        |
+      | Please specify the main development branch | [DOWN][ENTER] |
```

Comment:
I'm creating a PR to re-enable cucumber-lint which got lost when refactoring the build scripts out of bin